supabase/migrations/20250815170919_unique-quote-ids.sql

```diff
@@ -0,0 +1,9 @@
+CREATE UNIQUE INDEX cashu_receive_quotes_quote_id_key ON wallet.cashu_receive_quotes USING btree (quote_id);
+
+CREATE UNIQUE INDEX cashu_send_quotes_quote_id_key ON wallet.cashu_send_quotes USING btree (quote_id);
+
+alter table "wallet"."cashu_receive_quotes" add constraint "cashu_receive_quotes_quote_id_key" UNIQUE using index "cashu_receive_quotes_quote_id_key";
```

Comment:
could different mints have same quote it for something? I guess they theoretically could right?